### DISCUSSION TITLE
vue-flow-form available for Vue.js v3

### DISCUSSION THREAD

**Discussion by davymlc** [ditdot-dev/vue-flow-form#158] [Q&A] [answered]
Hello DitDot dev team,

Thanks very much for sharing this awesome work.
This is a great component! 🚀
Do you have any plan to make a version of Flow Form available for Vue.js version 3? 
Thanks again.

Davy

**Comment by spinn** [comment, marked as answer]
Hi Davy,

experimental support for Vue 3 is available in the feature/vue3-support branch available here:
https://github.com/ditdot-dev/vue-flow-form/tree/feature/vue3-support

This has not yet been thoroughly tested so if you find any issues, just let us know.

**Comment by EkaterinaVu** [comment]
Hi @davymlc , 
Vue 3 support is now available  - starting with v2.0.1 Vue Flow Form has migrated from Vue 2 to Vue 3.
